resources/containers/delete-confirmation.tsx

```diff
@@ -34,6 +37,11 @@ const DeleteConfirmation: React.FC<DeleteConfirmationModalProps> = ({
       className={modal_styles.modal}
       overlayClassName={modal_styles.overlay}
     >
+    {isLoading && (
+        <div className="absolute inset-0 bg-white bg-opacity-70 flex items-center justify-center">
+          <Spinner />
```

Comment:
We have a custom  Spinner component in resources, no need extra styling here

Reply:
Here actually I want the spinner to show on top of the confirm dialog, I removed the new svg spinner that I added and now I am using the existing one.